README.md

```diff
@@ -79,3 +68,20 @@ terraform init --upgrade
 ```console
 terraform apply
 ```
+
+## Help
+
+Have a feature request or found a bug? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.gg/bzA6uY7); we'll be happy to help you get started!
+
+## Contributing
+
+Instead of using the latest stable release, a local copy of the repository must be used.
+
+1. [Install Go 1.17+](https://golang.org/doc/install)
+2. Clone the repository & build the provider
+   ```console
+   git clone https://github.com/iterative/terraform-provider-iterative
+   cd terraform-provider-iterative
+   make install
+   ```
+3. Use `source = "github.com/iterative/iterative"` in your `main.tf` to use the local repository (`source = "iterative/iterative"` will download the latest release instead), and run `terraform init --upgrade`
```

Comment:
It feels like we need "Copyright" section. See DVC.

Reply:
I think I wrote a lot of the DVC readme, ha.

Not sure the same is needed here - already added the licence badge to the top of the readme & docs site (and removed the licence section) in this PR.

Reply:
It looks like a best practice in Readme. And also a bit safer. But I'm ok with either.

Reply:
ok re-added to bottom